OData/src/System.Web.OData/OData/Query/Expressions/AggregationBinder.cs

```diff
@@ -264,7 +296,7 @@ private Expression BindAccessor(SingleValueNode node)
                     return CreatePropertyAccessExpression(BindAccessor(propAccessNode.Source), propAccessNode.Property);
                 case QueryNodeKind.SingleValueOpenPropertyAccess:
                     var openNode = node as SingleValueOpenPropertyAccessNode;
-                    return Expression.Property(BindAccessor(openNode.Source), openNode.Name);
```

Comment:
That's what I thought; just wanted to confirm. Thanks.

---
In reply to: [115844405](https://github.com/OData/WebApi/pull/973#discussion_r115844405) [](ancestors = 115844405)